Atomic/syscontainers.py

```diff
@@ -2079,6 +2096,7 @@ def get_manifest(self, image, remote=False):
 
         imagebranch = SystemContainers._get_ostree_image_branch(image)
         commit_rev = repo.resolve_rev(imagebranch, True)
+        # todo: ask question about here
         if not commit_rev:
             raise ValueError("Layer not found: %s.  Please pull the image again" % imagebranch.replace("sha256:", ""))
```

Comment:
This part feels a bit weird here.  `commit_rev` itself is a tuple right? and for other cases when commit_rev, we all output an internal error: E.g: in `_check_system_ostree_image`: Is that intended behavior? 
```
 current_rev = repo.resolve_rev(imagebranch, True)
        if not current_rev:
            raise ValueError("Internal error for image: %s.  Please pull the image again" % imagebranch)
```

Reply:
Deferring to @giuseppe